pkg/dockerclient/shell.go

```diff
@@ -722,6 +721,17 @@ func (c *ShellClient) Load(inPath string) error {
 	return err
 }
 
+func (c *ShellClient) GetVersion(quiet bool) (string, error) {
+	runOptions := &cmdrunner.RunOptions{
+		SkipLoggingBeforeAfterExecute: quiet,
+	}
+	output, err := c.runCommand(runOptions, `docker version --format "{{json .}}"`)
+	if err != nil {
+		return "", errors.Wrap(err, "Failed to get docker version")
+	}
+	return output.Output, nil
+}
```

Comment:
?

Reply:
a "foo foo" command to ensure docker deamon is still alive